### ISSUE TITLE
Pull party affiliation for contacts

### ISSUE THREAD

**Issue by MainStreamDanny** [gpsimpact/relational_gotv#82]
We've decided we'd like to pull the party affiliation of the contacts when they are matched.

Display as:

"Brandi Fisher (R)"

**Comment by jakelowen** [comment]
@MainStreamDanny - for this issue I will display party affiliation only after they are matched. If you want party to also appear on voter search results please open another issue.

**Comment by MainStreamDanny** [comment]
No, this is fine, thanks.
